cmd/podman/common/specgen.go

```diff
@@ -47,6 +47,12 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 	if err != nil {
 		return err
 	}
+	if s.ResourceLimits == nil {
```

Comment:
Do we need logic to clear this for the rootless v1 case?

Reply:
we need to error out when some limits are specified.

So we need to carry the limits (avoiding to set any default) and raise an error from libpod if any limit is specified but we are rootless on cgroup v1